### PR TITLE
GSoC22 : Testsuite - 1.2

### PR DESCRIPTION
**Pull request by pathange-s**
The following pull request focuses to increase the test coverage of the currently implemented test suite for Neutralinojs CLI.

- [x]  Write tests for `neu plugins` command.
- [x]  Write tests for `neu version` command.
- [x]   Write tests for `neu update` command.
- [x]   Write tests for `neu build` command.

### CODE REVIEW

File: spec/misc.spec.js [hunk 1]
```diff
@@ -12,4 +15,39 @@ describe('Run misc neu commands', () => {
...
+            assert.ok(output.data.includes('Global') &&
+                output.data.includes('neu CLI: v'));
+            assert.ok(output.data.includes('Project') &&
+                output.data.includes('Neutralinojs binaries') &&
+                output.data.includes('Neutralinojs client') &&
+                output.data.includes('Project version'));
+        });
+    });
+    after(() => {
+        runner.run('rm -r test*');
```
Comment (shalithasuranga) [inline spec/misc.spec.js:51]: All looks great, but I have an idea to make this function call better. I think we can have the clean-up code in `runner.cleanup()` with Node.js functions (not Unix command-line like this). We can merge this after fixing this ;). I think we can use `.run()` only for neu command-lines.

Thanks :tada:

File: spec/runner.js [hunk 2]
```diff
@@ -25,6 +26,32 @@ function decodeUTF8(decode) {
     return decode ? decode.toString('utf8') : null;
 }
 
+function cleanup() {
+    try {
+        run('rm -rf test*');
```
Comment (shalithasuranga) [inline spec/runner.js:31]: Could you please use filesystem functions from fs? I think, someday we need to run this on Windows too ;)

Thanks :tada:

Comment (pathange-s) [reply spec/runner.js:31]: @shalithasuranga Will do. Also, can you please check your discord dm quickly? It's a little urgent man.

Comment (pathange-s) [reply spec/runner.js]: Will do.

Comment (pathange-s) [reply spec/runner.js]: @shalithasuranga Will do. Also, can you please checkout your discord for once?

**Review by shalithasuranga — APPROVED** [review]
Looks great now :) I will create and assign you some GitHub issues for some code tweaks. Seems like we are getting closer to the project completion ;)

**Pull request merged**